sublime-package.json

```diff
@@ -11,6 +11,7 @@
             "PluginConfig": {
               "properties": {
                 "dev_environment": {
+                  "deprecationMessage": "\"dev_environment\" has been moved into \"settings\".",
```

Comment:
Things that are deprecated should still work but I guess it's better to have some relevant message than just an error.

Are you planning an update message also to notify about this change? I think that would be a good idea.

Reply:
> Things that are deprecated should still work but I guess it's better to have some relevant message than just an error.

I don't have an idea how to make it work in a reasonable way. Because the new setting in `settings` will always available, I have no idea whether it's set by a user (which should be used directly) or comes from the default (which should fallback to try the deprecated one).

Reply:
update message added in [`c174e6b` (#70)](https://github.com/sublimelsp/LSP-pyright/pull/70/commits/c174e6badde4c8cf41bf822a3e3eb99d44b5692d)

Reply:
figured it out. [`fcb5f0c` (#70)](https://github.com/sublimelsp/LSP-pyright/pull/70/commits/fcb5f0c23c72d0dca43b85c830cbd2a3d1c8648f)